Modules/Loadable/Segmentations/SubjectHierarchyPlugins/qSlicerSubjectHierarchySegmentationsPlugin.cxx

```diff
@@ -262,12 +268,24 @@ bool qSlicerSubjectHierarchySegmentationsPlugin::reparentItemInsideSubjectHierar
     return false;
     }
 
+  vtkSlicerSegmentationsModuleLogic* sgmentationsModuleLogic =
+    (qSlicerCoreApplication::application() ? vtkSlicerSegmentationsModuleLogic::SafeDownCast(
+    qSlicerCoreApplication::application()->moduleManager()->module("Segmentations")->logic()) : nullptr);
+
+  qSlicerScriptedLoadableModuleWidget* segmentEditorWidget = dynamic_cast<qSlicerScriptedLoadableModuleWidget*>(qSlicerCoreApplication::application() ?
```

Comment:
This would be very fragile: Segment Editor widget may not have been even instantiated. There could be also multiple segment editor widgets in the application in many modules. Also, how the user would know that the current default terminology entry in one of the Segment Editor widgets in one of the modules controls the terminology used in this conversion?

Instead, we could get the default from the application settings (`Segmentations/DefaultTerminologyEntry` is a global application setting which can be specified in Segmentations section of the application settings). It would be also much easier to access this registry key than a module widget.